### DISCUSSION TITLE
How can I start sioyek with specific parameters?

### DISCUSSION THREAD

**Discussion by NailGit** [ahrm/sioyek#819] [General]
In my file pref_user.config I defined `custom_background_color 1. 0.996 0.812`  and in key_user.config I defined 
`toggle_custom_color cc`
`toggle_scrollbar <C-b>`
Is it possible to start Sioyek with a custom color and with a scrollbar as in config files? 
In fact, each time I have to click `cc` and `Ctrl+b` to apply a custom background color and start the scrollbar.

**Comment by ahrm** [comment]
See the documentation here: https://sioyek-documentation.readthedocs.io/en/latest/configuration.html#startup-commands .

**Reply by NailGit** [reply]
Oh, yes, thank you very much. You have foreseen everything.
